pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -3089,6 +3106,7 @@ scheduler_plugin_execution_duration_seconds_count{extension_point="PreEnqueue",p
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			resetMetrics()
+			resetPodInfos()
```

Comment:
what is increasing attempts in this case?

Could we recreate podInfos for every case, instead?

Reply:
Some of `test.operations` (`addPodUnschedulablePods`) do.

Reply:
> Could we recreate podInfos for every case, instead?

Given all `test.operands` refer`pInfos` / `pInfosWithDelay` like the following, that'd require a big change in the test implementation, which I want to avoid (at least, in this PR)


```go
			operands: [][]*framework.QueuedPodInfo{
				pInfos[:30], // Evern test case refers to the same pInfos. 
				pInfos[30:],
			},
```